mock/analysis.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable import/prefer-default-export */
+export const donutChart1 = 50;
+
+export const donutChart2 = 70;
+
+export const barGraph1 = [
+  [
+    { name: 'meta1', x: '5-10-2020', y: 100 },
```

Comment:
Hey Anisha, can we decrease the length of this file by importing these mock objects from a separate file, it would make it much easier to view and review

Reply:
I didn't know that I was commenting on the mock file itself, thanks Github :)